OpenRA.Mods.Common/Traits/GivesExperience.cs

```diff
@@ -19,8 +20,8 @@ class GivesExperienceInfo : ITraitInfo
 		[Desc("If -1, use the value of the unit cost.")]
 		public readonly int Experience = -1;
 
-		[Desc("Grant experience for team-kills.")]
-		public readonly bool FriendlyFire = false;
+		[Desc("Stance the attacking player needs to receive the experience.")]
+		public readonly Stance[] Stances = { Stance.Neutral, Stance.Enemy };
```

Comment:
Yes that also works, I just took this directly from `GivesBounty`.
I can change it but I'd like us to decide on a definitive way forward so this doesn't come up again.